mkosi/__init__.py

```diff
@@ -2883,7 +2887,7 @@ def set_autologin(args: CommandLineArguments, root: str, do_run_build_script: bo
 
     os.chmod(override_file, 0o644)
 
-    pam_add_autologin(root, "pts/0")
+    pam_add_autologin(root, device_prefix + "pts/0")
```

Comment:
Nit, we prefer python f-strings (`f"{device_prefix}pts/0"). Ditto for the other changes.

Reply:
fixed